experiments/dataSources/airtable/Airtable_Applicants.xml

```diff
@@ -1,66 +1,435 @@
-<skuidpage unsavedchangeswarning="yes" personalizationmode="server" showsidebar="true" showheader="true">
+<skuid__page unsavedchangeswarning="yes" personalizationmode="server" showsidebar="true" showheader="true">
 	<models>
-		<model id="Applicants" query="true" createrowifnonefound="false" datasource="Airtable" processonclient="true" verb="GET" payloadcontenttype="JSON" endpoint="&lt;Your_Base_Id&gt;/Applicants" pathtocontent="records" fieldtargetobjects="records">
+		<model id="Interviewers" query="true" createrowifnonefound="false" datasource="Airtable" verb="GET" payloadcontenttype="JSON" endpoint="<Your_Base_Id>/Interviewers" pathtocontent="records" fieldtargetobjects="records">
```

Comment:
All instances of `<Your_Base_Id>` in this page XML need to have the < > replaced with escaped sequences, or else Skuid will fail to parse this page XML when importing.
```suggestion
		<model id="Interviewers" query="true" createrowifnonefound="false" datasource="Airtable" verb="GET" payloadcontenttype="JSON" endpoint="&lt;Your_Base_Id&gt;/Interviewers" pathtocontent="records" fieldtargetobjects="records">
```